Doc/library/csv.rst

```diff
@@ -197,10 +197,10 @@ The :mod:`csv` module defines the following classes:
    Create an object which operates like a regular writer but maps dictionaries
    onto output rows.  The *fieldnames* parameter is a :mod:`sequence
    <collections.abc>` of keys that identify the order in which values in the
-   dictionary passed to the :meth:`writerow` method are written to file
+   dictionary passed to the :meth:`writerow<csvwriter.writerow>` method are written to file
```

Comment:
And we can do the same here:

```suggestion
   dictionary passed to the :meth:`~csvwriter.writerow` method are written to file
```

Sorry, I realise I suggested the longer version to you yesterday, and should have suggested this shorter one, but it's good to know both :)